source/Endpoints.rst

```diff
@@ -0,0 +1,126 @@
+|                    |                                                                               |                                                                                                                        |
+|                    |                                                                               | Note that you should implement either `/positions`_ or `/balances`_ or both endpoints in your integration.             |
++--------------------+-------------------------------------------------------------------------------+------------------------------------------------------------------------------------------------------------------------+
+| `/executions`_     | Gets a list of executions (fills or trades) for an account (sub account)      | Required when ``supportExecutions: true`` is set in `/accounts`_.                                                      |
+|                    | and an instrument.                                                            | If `/executions`_ is not implemented, transactions will not be displayed on the chart.                                 |
++--------------------+-------------------------------------------------------------------------------+------------------------------------------------------------------------------------------------------------------------+
+| `/ordersHistory`_  | Gets order history for an account (sub accounts).                             | Required when ``supportOrdersHistory: true`` is set in `/accounts`_. All :ref:`orders <trading-concepts-orders>`       |
+|                    |                                                                               | that come in response to ``/ordersHistory`` requests are displayed on the *History* tab.                               |
++--------------------+-------------------------------------------------------------------------------+------------------------------------------------------------------------------------------------------------------------+
+| `/previewOrder`_   | Gets estimated cost, commission, and other order information                  | Required when either ``supportPlaceOrderPreview`` or ``supportModifyOrderPreview`` is set to ``true`` in `/accounts`_. |
+|                    | without the order actually being placed or modified.                          |                                                                                                                        |
++--------------------+-------------------------------------------------------------------------------+------------------------------------------------------------------------------------------------------------------------+
+| `/quotes`_         | Gets current instrument prices and its data restrictions.                     | TradingView highly recommends implementing `/quotes`_ due to possible delays in data from the exchange.                |
```

Comment:
I wonder if it's correct to say `its` in this case because this pronoun relates to the attribute noun(not the main one). Couldn't find any rule for it, let me know if there is one. I'd suggest 
```suggestion
| `/quotes`_         | Gets the current instrument's prices and its data restrictions.                     | TradingView highly recommends implementing `/quotes`_ due to possible delays in data from the exchange.                |
```
or 
```suggestion
| `/quotes`_         | Gets the instrument's current prices and its data restrictions.                     | TradingView highly recommends implementing `/quotes`_ due to possible delays in data from the exchange.                |
```
depending on the meaning

Reply:
@anachuprina Could you find anything about this one?